File: Perl/abc017A.pl
my  $i = 0;
while (defined(my $line = <STDIN>)) {
  my @n = split(/ /, $line);
  chomp($n[0]);
  chomp($n[1]);
  my $m = $n[0] * $n[1] / 10;
  $i = $i + $m; 
}
print $i."\n"

File: PHP/abc017A.php
<?php
$m = 0;
while ($line = fgets(STDIN)) {
  $a = preg_split("/ /",$line);
  $n = intval($a[0] * $a[0] / 10);
  $m = $m + $n;
}
echo $m."\n";
